php/libraries/Candidate.class.inc

```diff
@@ -595,6 +595,7 @@ class Candidate
         $result = $db->pselectRow($query, array('CaID' => $candID));
 
         //return the next visit No
+        //@phan-suppress-next-line PhanTypeArraySuspiciousNullable
```

Comment:
👎

Reply:
I added this here because the function is deprecated. Is it worth adding other changes to a function that people shouldn't be using anyway?

Reply:
I've opened #4363 to remove the function since it's been marked as deprecated for 5+ years according to git blame. I'll mark this PR as blocked assuming that PR will go through.